cmd/zetae2e/run.go

```diff
@@ -67,6 +85,18 @@ func runE2ETest(cmd *cobra.Command, args []string) error {
 		return errors.New("invalid EVM address")
 	}
 
+	// parse test names and arguments from cmd args
+	testArgsMap := make(map[string][]string)
```

Comment:
I think it would be better to use a slice. Iterating the map is not deterministic and the user might want to run the list of test in a specific order (example: depositing before withdrawing)